wrapper/src/test/java/integration/container/tests/AdvancedPerformanceTest.java

```diff
@@ -97,8 +99,16 @@ public class AdvancedPerformanceTest {
 
   private static final ConcurrentLinkedQueue<PerfStat> perfDataList = new ConcurrentLinkedQueue<>();
 
-  protected static final AuroraTestUtility auroraUtil =
-      new AuroraTestUtility(TestEnvironment.getCurrent().getInfo().getAuroraRegion());
+  protected static final AuroraTestUtility auroraUtil;
+
+  static {
```

Comment:
I'd suggest to move this logic to a static method. It just adds readability to the code.